setup.cfg

```diff
@@ -1,12 +1,59 @@
 [metadata]
+name = oauthlib
+version = attr: oauthlib.__version__
+author = The OAuthlib Community
+author_email = idan@gazit.me
```

Comment:
we can use the email of two current active maintainer here

Reply:
@JonathanHuot Your call on changing the Author?